crates/tx5/src/config.rs

```diff
@@ -3,54 +3,99 @@ use tx5_connection::WebRtcConfig;
 use tx5_core::deps::serde_json;
 
 /// Tx5 endpoint configuration.
+#[derive(Clone)]
+#[non_exhaustive]
```

Comment:
This is a breaking change now, and this is the part I can't backport. I think it's worth doing now because it means we can add new config parameters without it being a breaking change. I wish we'd done it sooner so adding `webrtc_connect_timeout` wouldn't be a breaking change.